### PR TITLE
Add a contributing page

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
And link to cookbooks page in sidebar navigation

### CODE REVIEW

File: contributing.qmd [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+
+## How to contribute
+
+### 0. General
+
+Fork the repository, make changes, use `quarto preview` to make sure the changes look good and open a pull request.
+
+### 1. Adding a new format
+
+1. Create a folder with the formats name and, within that folder, an `intro.qmd`. The `intro.qmd` file should describe the basics about that format.
```
Comment (kylebarron) [inline contributing.qmd:26]: Maybe add a link to the quarto markdown syntax?

File: contributing.qmd [hunk 2]
```diff
@@ -1,10 +1,44 @@
 ---
```
[the remark this answers is not in the document]
Comment (abarciauskas-bgse) [reply contributing.qmd:1]: 🤔 I'm not sure I know enough about copyright or how it applies here, do we need it? Do we need to protect this content?

Yes authors should be listed, perhaps just in a list on the welcome page?

**Comment by abarciauskas-bgse** [comment]
I believe I have addressed comments. I'm not sure why the preview isn't working yet, but did notice that it's redirecting to devseed.com 🤔

**Comment by kylebarron** [comment]
The preview is cool! Does it ever get cleaned up? I.e. after the PR is merged does the bot push again to the `gh-pages` branch to delete the `/pr-preview/pr-33/` directory? Or do those preview directories just accumulate?

**Comment by abarciauskas-bgse** [comment]
For completeness: @kylebarron yes the tool https://github.com/marketplace/actions/deploy-pr-preview cleans up the pr-preview from gh-branches after the PR is merged

**Pull request merged**